src/TemplatesBuilder.js

```diff
@@ -164,10 +169,13 @@ export default class TemplatesBuilder {
 
     return {
       // This key will be unique for resource templates, property templates.
-      key: `${this.subjectTemplate.key} > ${propertyUri}`,
+      key: `${this.subjectTemplate.key} > ${Object.keys(propertyUrisMap)
+        .sort()
+        .join(", ")}`,
       subjectTemplateKey: this.subjectTemplate.key,
       label: this.valueFor(propertyTerm, rdfsLabel),
-      uri: propertyUri,
+      uris: propertyUrisMap,
```

Comment:
This is the change in model for property templates -- a property template can have multiple property URIs.

defaultUri below is just a shortcut to getting the first. In the future, may allow configuring this in template.